packages/go_router/CHANGELOG.md

```diff
@@ -1,5 +1,6 @@
 ## 6.3.0
 
+- Supports returning values on pop.
```

Comment:
Please publish a new version

Reply:
It looks like the conflict is back :(

Reply:
Yep, I'm resolving them rn

Reply:
Everything is a mess hahahaha, I'll rebase it to main and made the implementation again 🙃